example/src/App.tsx

```diff
@@ -15,6 +15,18 @@ import {PlatformInfo} from './PlatformInfo';
 // We don't need this workaround in New Expensify App since Reanimated is imported before Live Markdown.
 console.log(Animated);
 
+function handleFormatSelection(selectedText: string, formatCommand: string) {
+  if (formatCommand === 'formatBold') {
+    return `*${selectedText}*`;
+  }
+
+  if (formatCommand === 'formatItalic') {
+    return `_${selectedText}_`;
+  }
+
+  return selectedText;
```

Comment:
This is a nit but this is a perfect place to use `switch/case` block.
```suggestion
  switch (formatCommand) {
    case 'formatBold':
      return `*${selectedText}*`;
    case 'formatItalic':
      return `_${selectedText}_`;
    default:
      return selectedText;
  }
```